packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -189,15 +189,21 @@ describe('alerts api', () => {
     ]);
     expect(response).toEqual({});
   });
-});
 
-test('sort alerts', async () => {
-  const client: CogniteClient = setupLoggedInClient();
-  const response = await client.alerts.list({
-    sort: {
-      property: 'createdTime',
-      order: 'desc',
-    },
+  test('sort alerts', async () => {
+    const response = await client.alerts.list({
+      sort: {
+        property: 'createdTime',
+        order: 'desc',
+      },
+    });
+    expect(response.items.length).toBeGreaterThan(0);
+  });
+
+  test('test limit', async () => {
+    const response = await client.alerts.list({
+      limit: 1,
+    });
+    expect(response.items.length).toBe(1);
```

Comment:
question(blocking): How many alerts are already there ?

Reply:
I think we should create 2 alerts and then check for limit: 1 and see if it returns just 1.

Reply:
I'm creating 10 alerts in my latest commit @saagar-cognite